src/Test/MakerTestEnvironment.php

```diff
@@ -260,7 +260,7 @@ private function buildFlexSkeleton(): void
         }
 
         // fetch a few packages needed for testing
-        MakerTestProcess::create('composer require phpunit browser-kit symfony/css-selector --prefer-dist --no-progress --no-suggest', $this->flexPath)
+        MakerTestProcess::create('composer require phpunit:1.1.* browser-kit symfony/css-selector --prefer-dist --no-progress --no-suggest', $this->flexPath)
```

Comment:
`symfony/test-pack` 1.2 brings us PHPUnit 12 which the tests don't run with

Reply:
no, `phpunit` is an alias for `symfony/test-pack` (see https://github.com/symfony/recipes/blob/main/symfony/test-pack/1.0/manifest.json#L2 and https://packagist.org/packages/symfony/test-pack#v1.1.0)